packages/activemodel-adapter/tests/integration/active-model-serializer-test.js

```diff
@@ -341,3 +341,18 @@ test("extractErrors camelizes keys", function() {
     firstName: ["firstName not evil enough"]
   });
 });
+
+test('extractErrors extracts polymorphic records errors', function() {
+  var payload = {
+    errors: {
+      evil_minions: [
+        { name: ['required'] }
+      ]
+    }
+  };
+
+  var serializer = env.container.lookup('serializer:application');
+  serializer.extractErrors(env.store, MediocreVillain, payload, null); // id (last param) is not relevant here
+
+  ok(true);
```

Comment:
Also very correct, my bad. I didn't look very far and kept my original test that was flat out erroring. That is also fixed!